google-account-plugin/src/com/google/cloud/tools/intellij/stats/UsageTracker.java

```diff
@@ -33,4 +34,81 @@ void trackEvent(@NotNull String eventCategory,
       @Nullable String eventLabel,
       @Nullable Integer eventValue);
 
+  /**
+   * Returns a fluent API for pinging tracking events.
+   *
+   * <p>Example:
+   * {@code
+   *  trackEvent("com.google.cloud.tools")
+   *    .withAction("appengine.deployment")
+   *    .andLabel("flex")
+   *    .setValue(1)
+   *    .send();
+   * }
+   *
+   * @param category this is used to identify the client being used. It will be typically the same
+   *                 for all invocations from the same plugin.
+   * @return a fluent interface that allows the user to set the 'action' being tracked by the event
+   */
+  //TODO: Check all invocations fo track event and consider defaulting 'category' to a global value.
```

Comment:
Currently, it is always "com.google.cloud.tools" for the both plugins. So, as discussed, removing the parameter seems a good idea.